### PR TITLE
Closing tags for non-void elements in Admin UI

### LINKED ISSUE #2025: Edit Scheduled Events Modal does not allow specifying minutes (only hours)
**Describe the bug**
When I would like to change the start- or end time or the duration of a scheduled event in the Admin UI, I can only change the hour, not the minute.

Screenshot 2020-11-01 at 15 13 00

**To Reproduce**
1. Go to https://stable.opencast.org/ and schedule an event for later today
2. Click on a blue hamburger menu on event belonging to a scheduled event
3. Switch to the "Scheduling" tab
4. See there is only a way for changing the hour but not the minute

**Environment Information**
 - OS: macOS Catalina 10.15.7
 - Browser Firefox 81.0.2
 - Opencast version with minor release `r/9.x` (Opencast 9.0.0.SNAPSHOT - 72347dc)

### LINKED ISSUE #2657: Event list in scheduled event edit modal is empty
**Describe the bug**
Event list of affected and events that can't be edited not shown when editing multiple scheduled events.

https://user-images.githubusercontent.com/2311611/120080475-63c0e200-c0b9-11eb-8995-662035db6a8e.mp4

**To Reproduce**
1. Go to https://stable.opencast.org/admin-ng/index.html
2. Select multiple scheduled events
2. Click on Actions
3. Scroll down to "Edit Scheduled Events"

**Environment Information**
 - Browser Firefox
 - Opencast version with minor release 9-snapshot and 9.4

### PR DESCRIPTION
**Pull request by Rillke**
- Angular JS only supports `<void elements/>` from the HTML spec.

See
- angular/angular.js#1237
- https://stackoverflow.com/a/31724215/2683737

- Add static testing to CI to avoid silent future breakage
  - Test uses html-validate
    - https://html-validate.org/
    - https://gitlab.com/html-validate/html-validate

- Add a creator to WF status for a more complete mock server response

Resolves #2657
Resolves #2025
Based on/ Contains #2718

image
Screenshot from 2021-06-12 15-09-33

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/#development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

**Comment by JulianKniephoff** [comment]
Since @Rillke called me out in Matrix, I'll just take this and #2718. :wink:

File: modules/admin-ui-frontend/app/scripts/shared/partials/modals/edit-events-metadata-modal.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h2 translate="BULK_ACTIONS.EDIT_EVENTS_METADATA.CAPTION"></h2>
       </div>
       <div class="full-col">
         <div class="obj tbl-list">
-          <header translate="BULK_ACTIONS.EDIT_EVENTS_METADATA.REQUEST_ERRORS.TABLE.CAPTION" />
+          <header translate="BULK_ACTIONS.EDIT_EVENTS_METADATA.REQUEST_ERRORS.TABLE.CAPTION" ></header>
```
Comment (JulianKniephoff) [inline modules/admin-ui-frontend/app/scripts/shared/partials/modals/edit-events-metadata-modal.html]:
```suggestion
          <header translate="BULK_ACTIONS.EDIT_EVENTS_METADATA.REQUEST_ERRORS.TABLE.CAPTION"></header>
```

File: modules/admin-ui-frontend/app/scripts/shared/partials/modals/edit-events-modal.html [hunk 2]
```diff
@@ -257,9 +257,9 @@ <h2 translate="BULK_ACTIONS.EDIT_EVENTS.CAPTION"><!-- Template --></h2>
               <table class="main-tbl">
                 <thead>
                   <tr>
-                    <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.TYPE" />
-                      <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.PREVIOUS" />
-                        <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.NEXT" />
+                    <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.TYPE" ></th>
+                    <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.PREVIOUS" ></th>
+                    <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.NEXT" ></th>
```
Comment (JulianKniephoff) [inline modules/admin-ui-frontend/app/scripts/shared/partials/modals/edit-events-modal.html]:
```suggestion
                    <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.TYPE"></th>
                    <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.PREVIOUS"></th>
                    <th class="fit" translate="BULK_ACTIONS.EDIT_EVENTS.SUMMARY.NEXT"></th>
```

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Very cool, more automated checking is always great, thanks for this! :smile: 

I have a couple very minor stylistic nitpicks. If you could still resolve those until tomorrow that would be great!

**Comment by Rillke** [comment]
> If you could still resolve those until tomorrow that would be great!

@JulianKniephoff pressed some buttons and your suggestions went into a new commit.

**Review by JulianKniephoff — APPROVED** [review]
Unfortunately I totally forgot our new rules about PRs having to be discussed at a technical meeting first, before being merged. I wasn't technically allowed to merge #2718, either. This will have to wait for 9.7, unfortunately. :slightly_frowning_face:

**Comment by Rillke** [comment]
> Unfortunately I totally forgot our new rules about PRs having to be discussed at a technical meeting first, before being merged.

If we would like to get picky, we could say, while it applies to [newer versions](https://docs.opencast.org/develop/developer/development-process.html#acceptance-criteria-for-patches-in-different-versions), this requirement is [not yet documented for Version 9 of Opencast](https://docs.opencast.org/r/9.x/developer/development-process.html).

**Comment by JulianKniephoff** [comment]
Alright, let's be pragmatic about this. :smile:

**Pull request merged**